modules/experimental/pages/war-engine-barging.adoc

```diff
@@ -0,0 +1,48 @@
+= War engine barging
+
+Two experimental options to improve the rules for when you make a charge move with a War Engine unit.
+
+****
+.The problem
+Often, when you make a charge move with your War Engine unit, you'd like to move it into base contact with multiple enemy units.
+The rules do not actually explain how this should work. 
+
+.The proposal
+Define charge move rules that explain how you can move your War Engine unit into base contact with multiple enemy units.
+We base these options around the idea that a massive war engine should be able to barge past lesser units to some extent, to get deeper into an enemy detachment and so make contact with more units.
+****
+
+---
+
+When it charges, a war engine can barge some enemy units aside and plough deeper into the enemy line.
+
+So, when you make a charge move with a War Engine unit, use the procedure below.
+
+(But be consistent -- you and your opponent should both agree and use either option A or option B for your entire game.)
+
+NOTE: You may *not* barge aside enemy War Engine units.
+We consider them to be too massive and dangerous for this to be a viable manoeuvre.
+
+.Option A and B
+. As in the standard assault move rules, move your War Engine into base contact with the closest enemy unit.
+
+.Option A: Barging based on initial Damage Capacity (DC)
+[start=2]
+. Rather than stop there, you may continue your move if you have more movement available.
+. You may not continue your move past the point at which you run out of available movement or the War Engine has made contact with a number of enemy units equal to half its initial Damage Capacity -- whichever happens first.
+
+.Option B: Barging based on distance (5 cm)
+[start=2]
+. Rather than stop there, you may move your War Engine up to a further 5 cm, or the remainder of its available move -- whichever is the least.
+ * Remember that a War Engine unit can attack or be attacked by no more units than the number equal to half its initial Damage Capacity. So, you must end your assault move if you reach this condition before you run out of movement.
```

Comment:
```suggestion
So, you must end your assault move if you reach this condition before you run out of movement.
```

This isn't a pull request suggestion, but picking out a specific bit of text to comment on.

The original text for the 5 cm option doesn't limit the number of units that can be barged out of the way.

Is this an intentional change to the option? I wondered about adding this as a third restriction, but decided to leave it out to reduce complexity. The note was included as a reminder that even if more than DC/2 units end up in base contact, they won't all be fighting in close combat.

Reply:
>The original text for the 5 cm option doesn't limit the number of units that can be barged out of the way.

With respect, yes it did. The 5 cm option section ended with a note that included this text:

>_Remember war engine units can only attack or be attacked by units equal to half the war engine unit’s starting Damage Capacity._

I've just moved it up, integrated it with the procedure and reworded it a bit. The previous note that came after both procedures also took some parsing because it wasn't clear whether it related to only the last procedure, or both (and was in fact a mix because it also had "Enemy war engines *cannot* be barged aside.").

This was another reason why I created one procedure with two optional clauses rather than leave it as two very similar procedures with 'other stuff' near them where the relationship is less clear.

Reply:
> > The original text for the 5 cm option doesn't limit the number of units that can be barged out of the way.
> 
> With respect, yes it did. The 5 cm option section ended with a note that included this text:
> 
> > _Remember war engine units can only attack or be attacked by units equal to half the war engine unit’s starting Damage Capacity._

I might be missing a rule somewhere, in that case. I'm not aware of a direct link in E40k between how many units are allowed in base contact, and how many are allowed to attack/be attacked. 

It's Epic Armageddon that limits the number of units that can enter base contact.

Reply:
Possibly at cross purposes again, but...

While I only rewrote what you already wrote, it did sound familiar and I double checked it to be sure:

>Ordinary units can only be ganged up on by a maximum of two enemy units, but war engines can be attacked by a number of units equal to half of their starting Damage Capacity. Units attacking a war engine in close combat must be in contact with the war engine model or its base.

https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/WarEngines/WarEnginesInCloseCombat.htm

So it's not just EA, unless there is some nuance in there that you're referring to?

Reply:
Ah, you're talking about number in base contact vs number that are actually attacking. Hmmm... that seems a bit obtuse and needlessly messy.

Reply:
> Ah, you're talking about number in base contact vs number that are actually attacking. Hmmm... that seems a bit obtuse and needlessly messy.

That, and 'Add up Assault values':

> Each player starts by adding up the Assault values of units in base-to-base contact with the enemy. A maximum of two units can 'gang up' on each enemy unit.

I think there's a _very_ strong case for rewriting the main rules to limit it by base contact and moving that restriction to the charge moves section so that it's not being checked _after_ all the charge moves have been made, but we haven't done that yet and I wasn't sure if that was why you'd 'changed' the text.

Note that I don't mind the change, but I wasn't sure if it was because you thought the game logic be changed or because your brain had sensibly over-ridden the obtuseness of the GW text.

Reply:
>Note that I don't mind the change, but I wasn't sure if it was because you thought the game logic (edit:) should be changed or because your brain had sensibly over-ridden the obtuseness of the GW text.

Okay, I don't mean to labour the point, but you still seem to be under the misapprehension that I changed your meaning: That can happen, but as mentioned above, in this case I do not believe that it did — if there is a 'change', it stems from your preceding version in which you wrote:

>Remember war engine units can only attack or be attacked by units equal to half the war engine unit’s starting Damage Capacity.

Having cleared that up, I think that you are right that, strictly speaking, GW's text is ambiguous and could make people think that the number of units in base contact and the number that can actually attack/be attacked are two separate things. But, unless there is other evidence to corroborate it, I think that that is a pretty obtuse interpretation that rests on GW's poor wording rather than actual design intent.

Given how messy that interpretation appears to be, I agree that we should 'override' it (or rather, in my opinion, clarify so that there is no longer room for the erroneous interpretation).

Reply:
Per video chat, I did actually get double-blind on this one, so sorry about that.

We seem to agree that the other interpretation is a bit obtuse and messy, and that we should take the simpler interpretation in which there is a 1:1 relationship between base contact and number of attackers/targets. So, I'll resolve this conversation.